addons/mail/static/src/new/core/messaging.js

```diff
@@ -794,11 +803,28 @@ export class Messaging {
         Message.insert(this.state, messageData, message.originThread);
     }
 
+    toggleChannelFold(threadLocalId) {
+        if (this.env.isSmall) {
+            return;
+        }
+        const thread = this.state.threads[threadLocalId];
+        if (thread?.model === "mail.channel") {
+            return this.orm.silent.call("mail.channel", "channel_fold", [[thread.id]], {
+                state: thread.state,
+            });
+        }
+    }
```

Comment:
This method is not correct named. It gives the impression that calling it would toggle the fold state of channel, but actually it just asks the server to change the value (as long as it's in non-mobile).

For the moment I suggest rename to `notifyChatWindowState`, so that its name matches better what it actually does.